discovery/discovery.py

```diff
@@ -72,27 +72,4 @@ def add_file(self, path: str, reconstruct=False):
 
 
 if __name__ == "__main__":
-    # locally test the mock filesystem
-    launch_config = yaml.safe_load(open("launch_config.yaml"))
-    discovery_instance = Discovery(launch_config)
-    import os
-
-    if not os.path.exists('../test/mock_filesystem'):
-        raise "You forgot to create a mock filesystem!"
-
-    discovery_instance.add_files("../test/mock_filesystem")
-    discovery_instance.reconstruct_metadata()
-    discovery_instance.create_visual("test_visual")
-
-    from test.datagen import FakeDataGen
-
-    fake_data = FakeDataGen()
-    fake_files = fake_data.build_df_to_file(1000, "matcher_test", index_type="categoric", continuous_data=5,
-                                            file_spread=2)
-    discovery_instance.add_file(fake_files[0])
-    discovery_instance.add_file(fake_files[1])
-
-    dataframe_matcher = DataFrameMatcher(
-        discovery_instance.file_handler.loaded_files[fake_files[0]],
-        discovery_instance.file_handler.loaded_files[fake_files[1]])
-    dataframe_matcher.match_dataframes()
+    pass
```

Comment:
it might be better to keep this, this block won't affect end users, and can be used for easy local testing

Reply:
Isn't it exactly what we're supposed to do in the tests though? Why keep this and the test folder? I can name it something other than "unit" test.

Reply:
I prefer to keep it, it's easier to smash on the start button than to fiddle with tests

Reply:
I can write the exact same code block in a test, and then we can have multiple versions of it (e.g. Pearson + Levenshtein, or categorical + LCS) at the same time. In discovery.py we have to replace it constantly. I just don't know what to name it? What is a test called that doesn't have a hard pass/fail criterion?

But it doesn't matter. I'll move it back in discovery.py and move on.

Reply:
the block of code here is for debugging, not testing
like blaze said, its easy to just click to start button and see if your changes have been done correctly

these aren't unit tests, they only exist to see the result of your work, so its convenient to just change it to display whatever piece of data you want to see

Reply:
As a compromise to keep these not-tests and not overwrite each other's code, Balazs agreed to create a sandbox folder, similar to the test folder, and make our debugs in there, in separate files. Any output would be gitignored. Do you agree?

Reply:
I don't really care as long as it doesn't overcomplicate debugging